### PR TITLE
Add onboarding guide and checklist

### PR DESCRIPTION
**Pull request by mostafa-anm**

### CODE REVIEW

File: accounts.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+### Accounts
+
+Please always make sure to turn on 2-Step Verification. It is mandatory.
+
+#### Discord
+Discord is the main communication channel in our engineering team. please install discord on your machine and ask your team lead to give you the necessary permissions.
```
Comment (codingjoe) [inline accounts.md:6]: You could add a link to our discord, I believe there is a way to request access.

File: accounts.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+#### GitHub
+Please give your team lead your GitHub username. He or she will invite you to our organization or as a collaborator to a repository.
+
+Make sure to add your name and any picture (does need to be you) to help your new teammates to identify you quicker.
+
+#### Heroku
+We have Heroku Enterprise, please create an account on heroku and ask your team lead to give you the necessary permissions.
```
Comment (codingjoe) [inline accounts.md:14]:
```suggestion
We use Heroku Teams, please create an account on Heroku and ask your team lead to give you the necessary permissions.
```

We don't use enterprise ;)

File: onboarding.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+### Onboarding
+Congratulations and welcome to voiio!
+
+We are happy to have you in the team, we know that starting a new job can be both exciting and a little bit daunting. This guide is meant to help you get settled.
+
+### Checklist
+To make sure you get a smooth onboarding and enjoy it, your buddy will be there to help you accomplish the following checklist.
```
Comment (codingjoe) [inline onboarding.md:7]: You are referencing the buddy here, without prior mentioning it. Maybe you want to add a section about the buddy first?

File: onboarding.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+2. Fix a small technical issue using some technology you already know. This will be close to vanilla Python/Django or Javascript/Vue.js.
+
+3. A more challenging technical issue, this time you will get to work with a part of our stack that is new to you.
+
+4. Now you are ready to dive into our business domain and solve a substantive issue for one of our products and its users.
+
+#### First day
+* Setup computer and install the development environment.
+* Get access to all necessary [accounts](accounts.md).
+* Push your first commit to production.
```
Comment (codingjoe) [inline onboarding.md:22]:
```suggestion
* Deploy your first commit on production.
```

Comment (codingjoe) [reply onboarding.md:22]: The last task of the first day should be to extend or update the onboarding guide.

File: onboarding.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+#### First week
+* 15 min daily checkins with buddy.
+* Intro to Frontend with Sebastian.
```
Comment (codingjoe) [inline onboarding.md:26]: Maybe use GitHub handles here to link to the actual GH profile.

File: onboarding.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+* Intro to Metabase with Rust.
+* Intro to Google Analytics with Mos.
+
+#### First Month
+* We do a team event all the tech team together.
```
Comment (codingjoe) [inline onboarding.md:31]:
```suggestion
* We, the entire tech team, do your first team event together.
```

File: onboarding.md (hunk 4 above)
Comment (codingjoe) [reply onboarding.md:22]: Never mind, found it. However, I don't want people to forget about it. IMHO it should be an explicit first-day task.

**Comment by codingjoe** [comment]
FYI, I'd add an empty newline after each headline in MD.

File: accounts.md (hunk 2 above)
Comment (mostafa-anm) [reply accounts.md:14]: Hehe oh ok oops :)

File: accounts.md (hunk 1 above)
Comment (mostafa-anm) [reply accounts.md:6]: Hmm, I didn't find it. Seems to me that an invitation link needs to be shared with the new person. 

But I don't feel like putting a non expiring  invitation link publicly.

**Comment by mostafa-anm** [comment]
@codingjoe Thanks for the review, I did some adjustments in here.

**Pull request merged**